staging/src/k8s.io/apiserver/pkg/server/options/api_enablement.go

```diff
@@ -54,6 +59,7 @@ func (s *APIEnablementOptions) AddFlags(fs *pflag.FlagSet) {
 		"api/beta=true|false controls all API versions of the form v[0-9]+beta[0-9]+\n"+
 		"api/alpha=true|false controls all API versions of the form v[0-9]+alpha[0-9]+\n"+
 		"api/legacy is deprecated, and will be removed in a future version")
+	fs.StringVar(&s.EmulationVersion, "emulation-version", defaultEmulationVersion, "Emulation version for the apis.")
```

Comment:
We need to explain the flag more fully here.  I.e. assume this is all the information the cluster administrator has about the flag, try to provide a self contained summary of how it works, what values are allowed, and what they should expect to happen when it is set.